pkg/plugins/golang/options.go

```diff
@@ -158,13 +160,15 @@ func (opts Options) NewResource(c newconfig.Config) resource.Resource {
 		res.API = &resource.API{}
 	}
 
-	if opts.DoDefaulting || opts.DoValidation || opts.DoConversion {
+	if opts.DoDefaulting || opts.DoValidation || opts.DoConversion || (len(opts.Spoke) != 0 && len(opts.Hub) != 0) {
```

Comment:
`Do{X}` must be true for this branch to execute, regardless of whether spoke/hub are set.
```suggestion
	if opts.DoDefaulting || opts.DoValidation || opts.DoConversion {
```

Reply:
It may also be worth deprecating `--conversion`, since `--hub`/`--spoke` imply conversion.

Reply:
In this sense I would say that I would probably implicitly set the `doConversion` variable in the `Validate` subcommand method or `PreScaffold` in phase 1.5.